packages/select/src/select.vue

```diff
@@ -77,7 +77,7 @@
               :class="[selectSize ? `is-${ selectSize }` : '']"
               :disabled="selectDisabled"
               :autocomplete="autocomplete"
-              :style="{ 'flex-grow': '1', width: inputLength / (inputWidth - 32) + '%', 'max-width': inputWidth - 42 + 'px' }"
+              :style="{ 'margin-left': prefixWidth && !selected.length || tagInMultiLine ? `${prefixWidth}px` : null, 'flex-grow': '1', width: inputLength / (inputWidth - 32) + '%', 'max-width': inputWidth - 42 + 'px' }"
```

Comment:
Since you have been using string template, why not update `inputWidth - 42 + 'px'` and `inputLength / (inputWidth - 32) + '%'` with the syntax?